src/home.js

```diff
@@ -0,0 +1,14 @@
+const home = () => {
+  const div = document.createElement('div');
+  div.setAttribute('id', 'main-content');
+  const container = document.createElement('div');
+  container.classList.add('best-restaurant');
+  const h1 = document.createElement('h1');
+
+  h1.innerHTML = 'John Doe\'s sushi restaurant';
```

Comment:
Yeah that's true, this is because when I started the project like 3 weeks ago while I was waiting to be re-paired, I didn't know about textContent property, now that I came back to finish the project, after working with my coding partner in several projects, I forgot that I did use innerHTML 😅 , that's why.